src/components/mc/base/ucc_mc_base.h

```diff
@@ -70,11 +71,17 @@ typedef struct ucc_mc_config {
 } ucc_mc_config_t;
 extern ucc_config_field_t ucc_mc_config_table[];
 
+// TODO: should move to ucc_mc.h?
```

Comment:
@lappazos why moving to ucc_mc.h? mpool is a property of MC implementation not mc interface. i think it is fine here.

Reply:
@vspetrov  because from now on everyone that will use ucc_mc_alloc will require to use ucc_mc_buffer_header. this is an integral part of ucc_mc_alloc, which is declared in ucc_mc.h

Reply:
yes, the header struct should be moved